libs/dotnet-sdk/UpdateStrategies/IFeatureBoardUpdateStrategy.cs

```diff
@@ -0,0 +1,9 @@
+using Featureboard.DotnetSdk.Models;
+
+namespace Featureboard.DotnetSdk.UpdateStrategies;
+
+public interface IFeatureBoardUpdateStrategy
+{
+  public Task<FeatureConfiguration?> Get(string featureKey, CancellationToken cancellationToken);
```

Comment:
So the design of the typescript SDK (if we want to mirror) is this has `hooks` to hook into event lifecycles.

It has a `connect()` function, which creates the connection. Then it has a `onRequest` hook.

The update strategy then has the hook into the request pipeline, polling just returns a completed task. 

This model works well with the `live` strategy too

Reply:
Updated